src/sentry/static/sentry/app/views/organizationIntegrations/constants.tsx

```diff
@@ -1,6 +1,6 @@
-export const INSTALLED = 'Installed';
-export const NOT_INSTALLED = 'Not Installed';
-export const PENDING = 'Pending';
+export const INSTALLED = 'Installed' as const;
+export const NOT_INSTALLED = 'Not Installed' as const;
+export const PENDING = 'Pending' as const;
```

Comment:
@scefali Do we need type casting here?

Reply:
@NisanthanNanthakumar This isn't typecasting. This is called a const assertion (https://mariusschulz.com/blog/const-assertions-in-literal-expressions-in-typescript). This is so I can use the constants as both a type and a value.